library/core/src/result.rs

```diff
@@ -216,17 +216,14 @@
 //! [`?`] can only be used in functions that return [`Result`] because of the
 //! early return of [`Err`] that it provides.
 //!
-//! [`expect`]: enum.Result.html#method.expect
+//! [`expect`]: Result::expect
 //! [`Write`]: ../../std/io/trait.Write.html
 //! [`write_all`]: ../../std/io/trait.Write.html#method.write_all
 //! [`io::Result`]: ../../std/io/type.Result.html
 //! [`?`]: ../../std/macro.try.html
```

Comment:
This could use `crate::try!`. But the link looks wrong in the first place - `?` uses the `Try` trait, _not_ `core::try!`.
```suggestion
//! [`?`]: crate::ops::Try
```